cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -27,7 +27,7 @@
 "containers":[
     {
     "name": "etcd-container",
-    "image": "gcr.io/google_containers/etcd:3.0.3",
+    "image": "gcr.io/groovy-sentry-504/etcd:3.0.4",
```

Comment:
I guess that the way to proceed with this PR would be:
- first it needs to be LGTMed
- the from this PR I will build a new image and push to gcr
- then we will merge this PR

Or am I missing something?